<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QWidget" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(53, 53, 53);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1000</width>
     <height>511</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>53</red>
         <green>53</green>
         <blue>53</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 53, 53);
font: 14pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QWidget" name="tabRecord">
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <attribute name="title">
     <string>Record</string>
    </attribute>
    <widget class="QPushButton" name="btnRec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/botones/BotonRec.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/botones/BotonConf.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/botones/BotonCaptura.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupRecGrafica1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="title">
      <string>Grafica 1</string>
     </property>
     <widget class="QRadioButton" name="rBtnPDDRRec1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PDDR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnDEDPRec1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DEDP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDDRec1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDD</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnMapRec1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapa</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDARec1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>25</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDCTRec1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDCT</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="rBtnPSPRec2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>290</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="title">
      <string>Grafica 2</string>
     </property>
     <widget class="QRadioButton" name="rBtnPDDRRec2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PDDR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnDEDPRec2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DEDP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDDRec2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDD</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnMapRec2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapa</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDARec2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>25</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDCTRec2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDCT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWebView" name="WRecG1" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
    <widget class="QWebView" name="WRecG2" native="true">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabReplay">
    <attribute name="title">
     <string>Replay</string>
    </attribute>
    <widget class="QGroupBox" name="groupRepGrafica2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>290</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="title">
      <string>Grafica 2</string>
     </property>
     <widget class="QRadioButton" name="rBtnPDDRRep2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PDDR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnDEDPRep2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DEDP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDDRep2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDD</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnMapRep2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapa</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDARep2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>25</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDCTRep2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDCT</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupRepGrafica1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="title">
      <string>Grafica 1</string>
     </property>
     <widget class="QRadioButton" name="rBtnPDDRRep1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>PDDR</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnDEDPRep1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>DEDP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDDRep1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDD</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnMapRep1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>25</y>
        <width>82</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Mapa</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDARep1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>25</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDA</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rBtnFDCTRep1">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>FDCT</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnOpenFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>420</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/botones/BotonOpenfile.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>420</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 255);</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/botones/BotonPlay.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/botones/Botonpausa.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWebView" name="WRepG1" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
    <widget class="QWebView" name="WRepG2" native="true">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>30</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="url" stdset="0">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images/source.qrc"/>
 </resources>
 <connections/>
</ui>
